iai_apartment/urdf/kitchen.xarco

```diff
@@ -1649,4 +1649,94 @@
 		</link>
 
 	</xacro:macro>
+
+<!-- extras -->
+	<!-- Collision okay surface for cabinet1-->
+		<material name="white">
```

Comment:
Please put the cabinet insides next to the cabinets themselves, so they are easier to find.

Reply:
added in the new commit.